### ISSUE TITLE
Moving beyond "algorithmic bias is a data problem"

### ISSUE THREAD

**Issue by bowni** [SubstraFoundation/referentiel-evaluation-dsrc#136]
From @RomainGoussault:
> [Moving beyond "algorithmic bias is a data problem"](https://www.cell.com/patterns/fulltext/S2666-3899(21)00061-1)

Question it raises: should we add an evaluation element (2.4 ?) on this aspect?

**Comment by RomainGoussault** [comment]
For now, I don't have any idea on a good evaluation element on this.

**Comment by bowni** [comment]
We could start @RomainGoussault with something basic, around the idea of being aware that such bias causes exist (not blindsight oneself about it).

**Comment by bowni** [comment]
@SaboniAmine if I remember correctly you were aware of this already at first workshops back in 2019 - early 2020. Would you see a way to frame an evaluation element on that topic?
I'd say for now:

- being aware it exists
- follow progresses made on this topic
- tools to prevent it: too soon for that for now (we could improve/strengthen the element in a year from now)

@RomainGoussault would that make sense?

**Comment by SaboniAmine** [comment]
This is quite a hard question.
The only analogy I guess I understand relies on information extraction from text manipulation.
The TF-IDF principle relies that ideas & concepts importance in a text relies on _the number of appearance_ in a corpus. This metric choice can be a bias, even if it is not related to data itself, as it encourages an algorithm based on TF-IDF vectorial representation to focus on main concepts in a text, and ignore minorities. 
As far as I rembember, this was the bias exposed in the Doc2Vec algorithm I mentioned in the first workshops, but I can't find it online anymore.
Some articles mentioned that loss functions in neural nets can behave the same way, thus induce a bias which may not be present in data (rare events can be ignored even if the train distribution is well balanced and representative of the reality).

Wdyt ?

**Comment by bowni** [comment]
I am preparing a new evaluation element on that topic @RomainGoussault @SaboniAmine @ClementMayer @HeytemBou. I think it is really going to be a key topic of the next 2-3 years.

**Comment by bowni** [comment]
See #143 !
